_layouts/homepage-layout.html

```diff
@@ -104,6 +104,12 @@ <h5>Data Tracker</h5>
                 <i  class="fa button-icon fa-edit"></i>
               </div>
             </a>
+            <a class="nav-homepage col-md-4 col-xl-3 col-sm-6 p-1" href='https://austin.maps.arcgis.com/apps/webappviewer/index.html?id=a206bee097194382a00515a4cc9dd029'>
+              <div class="col homepage-button homepage-button-row-2 p-2">
+                <h5>Markings Work Orders</h5>
+                <i  class="fa button-icon fa-arrow-up"></i>
```

Comment:
use the fa-roads instead since thats what Diana is already using in the knack app